### DISCUSSION TITLE
how to override Setup view page from OrchardCore.Setup?

### DISCUSSION THREAD

**Discussion by scil** [OrchardCMS/OrchardCore#11660] [Q&A]
I want to override Setup page https://github.com/OrchardCMS/OrchardCore/blob/dd4159f52434e679651a3027f89c0c274051d9ca/src/OrchardCore.Modules/OrchardCore.Setup/Views/Setup/Index.cshtml

In admin theme `My.Theme.TheAdmin` and none-admin theme `My.Theme.Default`, I tried created view file like this
```
Views\
  OrchardCore.Setup\
    Setup\
      Index.cshtml
```

But both failed.

What did I miss?

**Comment by ns8482e** [comment]
You can't - you need to implement your own Setup module if you need customized setup experience. and you need to enable your setup module in host startup with  `.AddSetupFeatures("MyModule.MyCustomSetup");`
